cartridge/roles/coordinator.lua

```diff
@@ -2,22 +2,21 @@ local log = require('log')
 local fiber = require('fiber')
 local checks = require('checks')
 local errors = require('errors')
-local netbox = require('net.box')
 local membership = require('membership')
-local uri_lib = require('uri')
 
 local vars = require('cartridge.vars').new('cartridge.roles.coordinator')
 local topology = require('cartridge.topology')
 local confapplier = require('cartridge.confapplier')
+local stateboard_client = require('cartridge.stateboard-client')
 
 local AppointmentError = errors.new_class('AppointmentError')
 local CoordinatorError = errors.new_class('CoordinatorError')
-local NetboxConnectError = errors.new_class('NetboxConnectError')
 
 vars:new('membership_notification', membership.subscribe())
 vars:new('connect_fiber', nil)
 vars:new('topology_cfg', nil)
-vars:new('conn', nil)
+vars:new('client', nil)
+vars:new('session', nil)
```

Comment:
If "client" contains "session". I don't see reasons to store session in vars as it could be easily obtained via "client:get_session()".